File: cpp/Assignment 1/SET B/b2.cpp
#include<iostream>
using namespace std;
int main(){
    int num ,total;
    cout<<"Enter num:";
    cin>>num;
    for (int i=1;i<=num;i++){
        total=0;
        for (int j=1;j<=i;j++){   
            total=total+j ;
            cout<<j;
            if (j < i){
                cout << " + ";
            }
        }
        cout<<" = ";
        cout<<total<<" "<<endl;
    }
}

File: php/Assignment 3/SET B/b2.php
<!DOCTYPE html>
<html>

<body>
    <form method='post'>
        Enter string:
        <input type="text" name="string" id="num"><br>
        <input type="radio" name="choice" value="1">push<br>
        <input type="radio" name="choice" value="2">pop<br>
        <input type="submit" name="submit">
    </form>
    <?php
    $arr = array("Element1", "Element2");
    $string = $_POST['string'];
    $choice = $_POST['choice'];
    switch ($choice) {
        case 1:
            array_push($arr, $string);
            print_r($arr);
            break;
        case 2:
            echo "Popped Element is " . array_pop($arr);
            break;
        default:
            echo "something went wrong";
    }
    ?>
</body>

</html>